File: js/1.js
// Find the sum of all the multiples of 3 or 5 below 1000. <<<--->>>

var total = 0;
for (var i = 1; i <= 999; i++) {
    if ((i % 3 == 0) || (i % 5 == 0)) {
        total = total + i;
    }
}
console.log(total); // 233168

File: elixir/1.exs
# Find the sum of all the multiples of 3 or 5 below 1000. <<<--->>>

multiples = fn(n) -> (rem(n, 3) == 0) or (rem(n, 5) == 0) end

res = 1..999 
      |> Enum.filter(&(multiples.(&1))) 
      |> Enum.sum

IO.puts res # 233168